[Loopring 3.6] Add a` from` field to DepositRequested

dong77 opened the pull request.

Brechtpd commented:
> Also removed the 'tokenID' field.

I remember this being requested by the relayer team (maybe here: https://github.com/Loopring/team/issues/129), but I don't know anything more than that.

dong77 commented:
> > Also removed the 'tokenID' field.
> 
> I remember this being requested by the relayer team (maybe here: [Loopring/team#129](https://github.com/Loopring/team/issues/129)), but I don't know anything more than that.

reverted
